recipes/sentry-crashpad/all/conanfile.py

```diff
@@ -26,96 +32,86 @@ class SentryCrashpadConan(ConanFile):
     }
 
     short_paths = True
-    generators = "cmake"
-    _cmake = None
 
     @property
     def _is_mingw(self):
         return self.settings.os == "Windows" and self.settings.compiler == "gcc"
 
-    @property
-    def _build_subfolder(self):
-        return "build_subfolder"
-
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
-
     @property
     def _minimum_compilers_version(self):
         return {
-            "Visual Studio": "15" if tools.Version(self.version) < "0.4.16" else "16",
+            "Visual Studio": "15" if Version(self.version) < "0.4.16" else "16",
             "gcc": "6",
             "clang": "3.4",
             "apple-clang": "5.1",
         }
 
     def export_sources(self):
-        self.copy("CMakeLists.txt")
-        for patch in self.conan_data.get("patches", {}).get(self.version, []):
-            self.copy(patch["patch_file"])
+        export_conandata_patches(self)
 
     def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
-        if self.settings.os not in ("Linux", "Android") or tools.Version(self.version) < "0.4":
+        if self.settings.os not in ("Linux", "Android") or Version(self.version) < "0.4":
             del self.options.with_tls
 
     def build_requirements(self):
         if self._is_mingw:
-            self.build_requires("jwasm/2.13")
+            self.tool_requires("jwasm/2.13")
 
     def requirements(self):
-        self.requires("zlib/1.2.12")
+        self.requires("libcurl/7.86.0")
+        self.requires("zlib/1.2.13")
         if self.options.get_safe("with_tls"):
-            self.requires("openssl/1.1.1n")
+            self.requires("openssl/1.1.1s")
 
     def validate(self):
-        if self.settings.compiler.get_safe("cppstd"):
+        if self.info.settings.compiler.get_safe("cppstd"):
```

Comment:
This has recently changed with 1.54 and beta6 (maybe 7 its hard to keep track)

this is not longer blocked

```suggestion
        if self.settings.compiler.get_safe("cppstd"):
```